workflow-examples/src/main/java/com/redhat/parodos/examples/ocponboarding/task/dto/jira/JiraApprovalStatus.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 Red Hat Developer
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.parodos.examples.ocponboarding.task.dto.jira;
+
+public enum JiraApprovalStatus {
+
+	DONE, DECLINED
```

Comment:
Are there any other possible statuses here?

Reply:
yes, it can also be 'Waiting for approval'.  But it will not be referenced as it will go to 'else' block to continue the checker: https://github.com/parodos-dev/parodos/blob/mvp-demo/workflow-examples/src/main/java/com/redhat/parodos/examples/ocponboarding/checker/JiraTicketApprovalWorkFlowCheckerTask.java#L98